### DISCUSSION TITLE
Notifications get highlighted after hovering

### DISCUSSION THREAD

**Discussion by moritztim** [community/community#116876] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

On the [Notifications page](https://github.com/notifications) there's some unexpected behavior when hovering over notifications.
Upon loading the page, this is what a section of it looks like:
![Before hover](https://github.com/community/community/assets/90388353/037e4ff8-3c9a-4b4e-b2dd-94eae203c569)
While I hover over a notification the background turns gray:
![During hover](https://github.com/community/community/assets/90388353/ba678012-6a1e-4efa-8470-318cc5923b84)
However, when I remove the cursor from the notification group (by which I mean the area of this screenshot), it becomes highlighted in blue with a line to the left:
![After hover](https://github.com/community/community/assets/90388353/c575b978-47d0-4a9d-afbe-021c29d762a0)

I don't know if this is intentional but either way, the following has happened to me countless times:
1. I open the notifications tab
2. I happen to move my mouse onto a notification
3. I switch to a different application
4. I go back and now this random notification is highlighted, as if it was new or unread.

**Comment by arthurauffray** [comment]
Hi there, I just tried this out and can't recreate it!

It might be a cache issue with your browser; try a `CTRL/command + SHIFT + R` to reload the page, ignoring cached content.
You could go even further and clear all stored GitHub data from your browser: `chrome://settings/content/siteDetails?site=https://github.com`

Otherwise, check any browser or system updates that could be causing this.

Hope this helps!

**Reply by moritztim** [reply]
I deleted all storage using firefox's storage tap in the dev tools. I can still recreate it but I've noticed my description is not quite right, it only happens when exiting a repository's notification "group", as explained in my most recent edit

**Reply by moritztim** [reply, marked as answer]
I can't reproduce in google chrome. I have reported it to mozilla now.

**Comment by arthurauffray** [comment]
Likely a browser thing then, try force quitting firefox, then try update it
or re install.

**Reply by moritztim** [reply]
This issue has persisted over multiple reboots and firefox updates.
